test/python/expected/sql_validation_framework/expected_drop.out

```diff
@@ -2,10 +2,10 @@ Unexpected drop found for cast  in file babelfishpg_common--1.0.0--1.1.0.sql
 Unexpected drop found for cast  in file babelfishpg_common--1.0.0--1.1.0.sql
 Unexpected drop found for cast  in file babelfishpg_common--1.0.0--1.1.0.sql
 Unexpected drop found for cast  in file babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for object sys.+ in file babelfishpg_common--1.1.0--1.2.0.sql
 Unexpected drop found for object sys./ in file babelfishpg_common--1.1.0--1.2.0.sql
 Unexpected drop found for object sys./ in file babelfishpg_common--1.1.0--1.2.0.sql
 Unexpected drop found for object sys./ in file babelfishpg_common--1.1.0--1.2.0.sql
-Unexpected drop found for object sys.+ in file babelfishpg_common--1.1.0--1.2.0.sql
```

Comment:
Whoops, sorry this was a mistake, but I think expected_drop.out from the artifacts might differ between runs (from assumption) but it seems the validation step only cares if the line exists or not (again, from assumption so I could be wrong). I removed this change since it is not needed.